### ISSUE TITLE
Invalid format of big numbers

### ISSUE THREAD

**Issue by kidar2** [adamwdraper/Numeral-js#666]
numeral(250212534625585000000).format("0,0.[00]") => "250 212 534 625 584 971 776"

**Comment by songchengen** [comment]
i have similar issue

![image](https://user-images.githubusercontent.com/34732107/71613461-a723d200-2be1-11ea-960e-48abed786b4c.png)

**Comment by kidar2** [comment]
So, it's not a bug, see 
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER

**Comment by lothisoft** [comment]
Fair enough!  How about formatting a number smaller than Number.MAX_SAFE_INTEGER?
numeral(999999999999).format("0,0.00000000"); => 999,999,999,998.99987793

See sandbox:
https://codesandbox.io/s/objective-spence-venj5?fontsize=14&hidenavigation=1&theme=dark
